language/samples/v1/samplegen_map_field_access.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
```

Comment:
Good catch, [this comes from the common template](https://github.com/googleapis/gapic-generator/blob/master/src/main/resources/com/google/api/codegen/py/common.snip#L36) for all .py files generated by the monolith
/cc @busunkim96

Is Python 2 support being dropped _with_ the µgenerator or _before_ the µgenerator?

This will probably disappear once we move over to samples from the µgenerator, good catch! /cc @software-dov

Reply:
Python 2 support is being dropped with the transition to the microgenerator.

Reply:
As this will fix itself with the transition to the µgenerator, I'm hiding this thread.

**It's a good catch!** But it comes from the common template which the monolithic generator uses for all Python files, not just code samples.